client/src/cmdhfmf.c

```diff
@@ -4084,15 +4093,15 @@ static int CmdHF14AMfEKeyPrn(const char *Cmd) {
 
     CLIParserContext *ctx;
     CLIParserInit(&ctx, "hf mf ekeyprn",
-                  "Download and print the keys from emulator memory",
-                  "hf mf ekeyprn --1k --> print MFC 1K keyset\n"
-                  "hf mf ekeyprn -w   --> write keys to binary file"
+                  "View in use keys from emulator memory",
+                  "hf mf ekeyprn --1k --> View MIFARE 1K keys from emulator memory\n"
```

Comment:
feels a bit cake on cake  explaining  _keys from emulator memory_  in the description of samples.
```
View all keys
View MIFARE 1K keys
```